build_library/sysext_prod_builder

```diff
@@ -30,12 +30,14 @@ create_prod_sysext() {
   local base_sysext="$4"
   local install_root="$5"
   local name="$6"
-  local grp_pkg="$7"
+  local grp_pkgs="$7"
   local pkginfo="${8:-}"
 
   local -a build_sysext_opts=()
 
-  local msg="Installing ${grp_pkg}' in sysext ${name}.raw"
+  local grp_pkg="${grp_pkgs//&/ }"
+
+  local msg="Installing ${grp_pkg} in sysext ${name}.raw"
```

Comment:
You're not annoying, quite the opposite - I really enjoy the feedback in your reviews because there's always something to learn, and the improvements are quite elegant!